shared/components/modals/WithdrawModal/WithdrawModal.js

```diff
@@ -44,13 +44,17 @@ export default class WithdrawModal extends React.Component {
   constructor(data) {
     super()
 
+    const { data: { currency } } = data
+
     this.state = {
       isShipped: false,
       address: '',
       amount: '',
       minus: '',
+      balance: null,
       ethBalance: null,
-      tokenFee: false,
+      isEthToken: helpers.ethToken.isEthToken({ name: currency.toLowerCase() }),
+      currentDecimals: constants.tokenDecimals[currency.toLowerCase()],
```

Comment:
```suggestion
???
      currentDecimals: constants.decimals[currency.toLowerCase()],
```

а если это не токен?

Reply:
Если не токен, то тоже было бы неплохо придерживаться максимального decimals. Тем более в том конфиге есть все монеты + токены, так что проблем не будет.